languages/ruby/pry_ruby.c

```diff
@@ -46,4 +46,6 @@ void pry_eval_file(const char *file)
     {
         state = ruby_exec_node(node);
     }
+
+    return state;
```

Comment:
we need to explicitly initialize L44 as `int state = 0;`. otherwise it _may_ be Undefined Behavior.